frontend/src/static/js/api.js

```diff
@@ -138,96 +138,101 @@ window.getGroupName = function getGroupName(group, filterGroupSelection) {
 };
 
 window.api = {
-  loadJSON(fname) {
-    if (REPORT_ZIP) {
-      const zipObject = REPORT_ZIP.file(fname.slice(2));
```

Comment:
I realized that `.slice(2)`  actually removes `./` from the file name, however that is unintuitive as the REPORT_DIR is explicitly defined as a const, and so I changed the implementation slightly.